Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs

```diff
@@ -1106,11 +1121,29 @@ private void internalEventHandler(object sender, ManagerEvent e)
 					#endregion
 
 					case 62:
-						if (ConnectionState != null)
-						{
-							ConnectionState(this, (ConnectionStateEvent)e);
-							return;
-						}
+						if (Connect != null)

```

Comment:
Well, "Connect" should be "Connected". This event just means we just connected OK to Asterisk.
Later if there is a "ReloadEvent", "ConnectEvent" won't be fired, but a "ReloadEvent".